README.md

```diff
@@ -14,13 +14,23 @@ The library can be loaded from SQLite as such:
 
 ### Docker
 
-A Docker image has been provided to simplify integrating the library into larger projects. When run, an interactive SQLite session is started with the library pre-loaded for testing.
+A Docker image has been provided containing the library statically linked against its dependencies, to simplify integrating the library into larger projects. An additional image is provided as an example of its usage.
 
-A Docker image with the library pre-built is available on the GitHub Container Registry (soon) to speed up building and simplify integrating into larger projects. If you prefer, you can also build the Docker image yourself:
+- `docker/lib/Dockerfile` builds a minimal image containing just the library. It cannot be run.
+- `docker/demo/Dockerfile` builds the library and starts an interactive SQLite demo session with it loaded.
 
-Run the following command:
+Both images are provided as packages on the GitHub Container Registry (`ghcr.io/uoy-research/rdkit-sqlite:main` and `ghcr.io/uoy-research/rdkit-sqlite-demo:main`). If you prefer, you can also build them yourself.
 
-`$ docker build -t 'rdkitsqlite' .`
+Run the following command to build the library image:
+
+`$ docker build -t 'rdkitsqlite' -f docker/lib/Dockerfile .`
```

Comment:
Can you change the tag to `rdkit-sqlite` to be consistent with how it's called in the repo